LoopKitUI/Extensions/UIFont.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  UIFont.swift
+//  LoopKitUI
+//
+//  Created by Nathaniel Hamming on 2020-01-17.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+extension UIFont {
+    public static var titleFontGroupedInset: UIFont {
+        return UIFont.systemFont(ofSize: 28, weight: UIFont.Weight.semibold)
```

Comment:
This does not respond to dynamic type. Perhaps take a look at https://developer.apple.com/documentation/uikit/uifont/1619030-preferredfont